### DISCUSSION TITLE
Simple Inverse One To One Relationship throws error "related is undefined"

### DISCUSSION THREAD

**Discussion by pixelmusik** [CodeDredd/pinia-orm#1788] [Q&A]
When implementing a simple one-to-one relationship (e.g. user has phone) everything works fine. But when I add the inverse relationship on the related model (e.g. phone belongs to user) as described in the official documentation, I am getting a JS error in the console.

"Uncaught (in promise) TypeError: related is undefined"

It seems that it is impossible to have an inverse relationship along with the one-to-one relationship. I can have either a hasOne relationship on the original model or a belongsTo relationship on the related model but not both? My two models "User" and "Phone" look exactly the same as described in the documentation: https://pinia-orm.codedredd.de/guide/relationships/one-to-one

What am I doing wrong?

- vue: v3.3.9
- pinia: v2.1.3
- pinia-orm: v1.7.2

**Comment by CodeDredd** [comment]
Please provide a reproduction 🙏
